Documentation/config.md

```diff
@@ -0,0 +1,289 @@
+---
```

Comment:
Is both config.md and config.adoc are valid? I see that only config.adoc being generated.